### PR TITLE
Remember last selected user in the StubIdp

### PR DESCRIPTION
**Pull request by albinsunnanbo**
Save selected user in a cookie, restoring last selected user when returning to the StubIdp.
Keeping a separate cookie for each sub-IDP to make them independent if you use more than one StubIdp.

I wrote this code as a test to see if it's useful or not. I can't really come to a conclusive decision.
On one hand it saves a few clicks/keystrokes when you mostly use your favorite user all the time, but on the other hand it's more complicated to use when you want to change user, especially since I moved focus to the Submit button if restoring the selected user was successful.

### CODE REVIEW

**Comment by albinsunnanbo** [comment]
@banansson or @hsahlberg may have an opinion?

File: Kentor.AuthServices.StubIdp/Scripts/ViewIndex.js [hunk 1]
```diff
@@ -98,4 +100,21 @@
             resetUnobtrusive($("form"));
         }
     });
+
+    var cookiedName = 'stubIdp.username';
```
Comment (AndersAbel) [inline Kentor.AuthServices.StubIdp/Scripts/ViewIndex.js]: Why the 'd' in the name of the cookie?

**Comment by AndersAbel** [comment]
I think that this makes sense, so I'm willing to merge it.

File: Kentor.AuthServices.StubIdp/Scripts/ViewIndex.js (hunk 1 above)
Comment (albinsunnanbo) [inline Kentor.AuthServices.StubIdp/Scripts/ViewIndex.js]: Just making sure you review the code properly.

**Comment by albinsunnanbo** [comment]
Fixed the spelling

**Comment by banansson** [comment]
Late reply, but I believe this would actually save us a few keystrokes more then be disruptive. I am all for.

**Comment by AndersAbel** [comment]
@banansson You're late to the party. It's already merged - and should be live on http://stubidp.kentor.se as it uses continuous delivery.

**Pull request merged**